README.md

```diff
@@ -0,0 +1,48 @@
+# hsac-fitnesse-fixtures
+[![Build Status](https://travis-ci.org/fhoeben/hsac-fitnesse-fixtures.svg?branch=master)](https://travis-ci.org/fhoeben/hsac-fitnesse-fixtures) [![Maven Central](https://img.shields.io/maven-central/v/nl.hsac/hsac-fitnesse-fixtures.svg?maxAge=300)](http://search.maven.org/#search|ga|1|g:"nl.hsac")
```

Comment:
Shouldn't the maven search url use: `a:"hsac-fitnesse-fixtures"`, instead of searching by group?